telco_cables_utility.test.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+import telco_cable_pairs_utility
+
+class TeleColorsTest(unittest.TestCase):
+    def test_number_to_pair(self):

```

Comment:
Nice separation of unit tests :)
See if you can think of test-names that include their expectations. Do additional test-possibilities emerge?